string text
string appID
string speakerName
---
string result
string fail_reason
